src/styles/ListCatalog.scss

```diff
@@ -1,5 +1,7 @@
 @import "@material/list/mdc-list";
 @import "@material/list/mixins";
+@import "@material/radio/mdc-radio";
```

Comment:
Import of these not required here, since it is already imported in `[Checkbox|Radio]Catalog.scss`. This'll result in double import.